### PR TITLE
Added avgbitrate. avgfps stats

### PR DESCRIPTION
**Pull request by auriv**
- Added avgbitrate/avgfps stats which now show average bitrate/fps since start of streaming.
- Fixed fps/bitrate stats to now show current fps/bitrate since last fps or bitrate change.
- the log now looks as:
frame= 1099 **avgfps= 16 fps= 14** q=0.0 size=   10221kB time=00:01:07.71 **avgbitrate=1236.6kbits/s bitrate=1018.2kbits/s** dup...

### CODE REVIEW

File: fftools/ffmpeg.c [hunk 1]
```diff
@@ -1770,11 +1786,22 @@ static void print_report(int is_last_report, int64_t timer_start, int64_t cur_ti
...
-
+/*Proximie*/
+#if 1
+            float cur_fps;
+            tr = ost->fps_reset_time ? (cur_time-ost->fps_reset_time) / 1000000.0 : t;
+            frame_number = ost->total_frames;
+            fps = t > 1 ? frame_number / t : 0;
+            cur_fps = tr > 1 ? ost->frame_number / tr : -1;
+            av_bprintf(&buf, "frame=%5d avgfps=%3.*f fps=%3.*f q=%3.1f ",
+                     frame_number, fps < 9.95, fps, cur_fps>0.0?cur_fps < 9.95:fps < 9.95, cur_fps>0.0?cur_fps:fps, q);
```
Comment (richardscreene) [inline fftools/ffmpeg.c:1797]: This looks wrong to me: There's 4 format specifiers but 5 following arguments.
Also, doesn't `fps < 9.95` resolve to a boolean?

File: fftools/ffmpeg.c [hunk 2]
```diff
@@ -1753,6 +1761,14 @@ static void print_report(int is_last_report, int64_t timer_start, int64_t cur_ti
     if (total_size <= 0) // FIXME improve avio_size() so it works with non seekable output too
         total_size = avio_tell(oc->pb);
 
+/*Proximie*/
+#if 1
+    tr = output_files[0]->bitrate_reset_time ? (cur_time-output_files[0]->bitrate_reset_time) / 1000000.0 : t;
+    reset_size = total_size - output_files[0]->reset_size;
+    cur_bitrate = tr>1 && reset_size >= 0 ? reset_size * 8 / tr / 1000.0 : -1;
```
Comment (richardscreene) [inline fftools/ffmpeg.c]: Pedantic: Can we be more consistent with whitespace?
```suggestion
    cur_bitrate = tr > 1 && reset_size >= 0 ? reset_size * 8 / tr / 1000.0 : -1;
```
(There are other occurrences too)

File: fftools/ffmpeg.c (hunk 1 above)
Comment (auriv) [reply fftools/ffmpeg.c:1797]: This is ffmpeg coding... I had to look it up too, the ".*" in the format specifier tells that number of spaces after period is specified in the argument list, which translates to 0 or 1 in this case

Comment (richardscreene) [reply fftools/ffmpeg.c:1797]: Fair enough!!

**Review by richardscreene — APPROVED** [review]

**Pull request merged**